guides/10_Video_Processor.md

```diff
@@ -188,42 +206,70 @@ class VideoLoadImageProcessor implements VideoFrameProcessor {
 }
 ```
 
-## Video Processing Usage
-### Custom processor usage during meeting
+## Additional Video Processing Use-Cases
+
+### Custom processor usage during meeting preview
+
+Local video post processing can be previewed before transmitting to remote clients just for a normal device.
 
 ```typescript
 import {
   DefaultVideoTransformDevice
 } from 'amazon-chime-sdk-js';
 
 const stages = [new VideoResizeProcessor(4/3)]; // constructs  processor
-
+const videoElement = document.getElementById('video-preview') as HTMLVideoElement;
```

Comment:
README and other examples are in JavaScript. I understand we specified `typescript` for all code blocks in this doc but I recommend writing samples for JavaScript.

Reply:
Sounds reasonable to me! will change.